.github/workflows/release.yml

```diff
@@ -28,13 +28,13 @@ jobs:
       - name: Run Lerna Bootstrap
         run: npm run bootstrap
       - name: Lint
-        run: npm run ci:lint
+        run: npm run lint
```

Comment:
Running `npm run lint` vs. `npm run ci:lint` will run linting for _all_ packages vs. just the ones that were changed since `origin/master`. However, when a PR is merged to `master`, it's commit is even with `origin/master` so the commands weren't being run.

I'm opting to run these commands on merge to `master` for all packages in the monorepo, which is fine for now since there's only 3 but we may want to come back and update if releases start taking longer as this monorepo grows.

Similar for the other below commands, too.